modules/tinymce/CHANGELOG.md

```diff
@@ -23,12 +25,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add-ons such as plugins and themes are no longer constructed using the `new` operator #TINY-8256
 - A number of APIs that were not proper classes, are no longer constructed using the `new` operator #TINY-8322
 - Moved the `print` plugin's functionality to TinyMCE core #TINY-8314
+- The `primary` property on dialog buttons has been deprecated. Use the new `buttonType` property instead #TINY-8304
 
 ### Fixed
 - The object returned from the `editor.fire()` API was incorrect if the editor had been removed #TINY-8018
 - The `editor.selection.getContent()` API did not respect the `no_events` argument #TINY-8018
 - The `GetContent` event was not fired when getting `tree` or `text` formats using the `editor.selection.getContent()` API #TINY-8018
 - The `table` plugin would sometimes not correctly handle headers in the `tfoot` section #TINY-8104
+- The height of `bar` dialog components is now 60px to make room for `sizeinput` components #TINY-8304
```

Comment:
This doesn't seem right as this was reverted wasn't it? Either way we definitely can't do this because it'll cause a number of issues elsewhere.

Reply:
As an example, this is what 60px does to the permanent pen dialog:
![image](https://user-images.githubusercontent.com/1575550/144972674-09c5640d-421e-4599-a59f-f153f19a1a86.png)

Reply:
yeah, I reverted a bunch of things and forgot some of the changelog entries.